thoth/user_api/api_v1.py

```diff
@@ -820,6 +827,33 @@ def _do_schedule(parameters: dict, runner: typing.Callable, **runner_kwargs):
     return {"analysis_id": runner(**parameters, **runner_kwargs), "parameters": parameters, "cached": False}, 202
 
 
+def _send_schedule_message(message_contents: dict, message_type: MessageBase):
+    message_contents["service_version"] = service_version
+    message_contents["component_name"] = component_name
+    message = message_type.MessageContents(**message_contents)
+    p.produce(message_type().topic_name, value=message.dumps())
+    if "job_id" in message_contents:
+        return (
+            {
+                "message_topic": message_type().topic_name,
+                "analysis_id": message_contents["job_id"],
+                "parameters": message_contents,
+                "cached": False,
+            },
+            202,
+        )
+
+    return (
+        {
+            "message_topic": message_type().topic_name,
+            "analysis_id": "job_id was not set",
```

Comment:
Do we want to raise an error instead to be more error prone? This should not happen.